### PR TITLE
build elf-sims docker image

### PR DESCRIPTION
**Pull request by wakamex**
build docker image containing latest version of `elf-simulations` code, built on top of the existing `python-base` docker image.

### CODE REVIEW

File: Dockerfile-elf-sims [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# copy hyperdrive contracts from migrations image
+COPY --from=migrations /src/ ./hyperdrive_solidity/
+# copy foundry over from migrations image
+COPY --from=migrations /usr/local/bin/ /usr/local/bin
+
+# compile contracts
+RUN ape compile
+
+# install elf-simulations
+RUN python -m pip install -e .
```
Comment (dpaiton) [inline Dockerfile-elf-sims:28]: why is it that you do not need to run pip install -r requirements.txt first?

Comment (dpaiton) [reply Dockerfile-elf-sims:28]: ohh I see it happens in the other file

**Review by dpaiton — APPROVED** [review]
very cool! Can't wait to spin these up

File: Dockerfile-elf-sims (hunk 1 above)
Comment (wakamex) [reply Dockerfile-elf-sims:28]: pulled in freshly built and dockerized from `python-base`! measure twice, cut once!

**Pull request merged**